pkg/cmd/accept/exec.go

```diff
@@ -118,16 +118,18 @@ func (o *Options) approveCSR(kubeClient *kubernetes.Clientset, clusterName strin
 	}
 	var csr *certificatesv1.CertificateSigningRequest
 	for _, item := range csrs.Items {
-		//Does not have the correct name prefix
-		if !strings.HasPrefix(item.Spec.Username, userNameSignatureBootstrapPrefix) &&
-			!strings.HasPrefix(item.Spec.Username, userNameSignatureSA) {
-			continue
-		}
-		//Check groups
-		groups := sets.NewString(item.Spec.Groups...)
-		if !groups.Has(groupNameBootstrap) &&
-			!groups.Has(groupNameSA) {
-			continue
+		if !o.skipApproveCheck {
```

Comment:
/approve

hrm, I think we need some refactor here. It seems we only approve only csr in the list, but we probably need to approve them all. Would you add a TODO before line 121 on we need to approve all CSRs that meets the check, and if the check is skipped, just approve all csrs in this list?